ui/pages/desktop-pairing/desktop-pairing.component.js

```diff
@@ -104,56 +100,71 @@ export default function DesktopPairingPage({
-        data-testid="desktop-pairing-otp-content"
-      >
+      <>
+        <Text variant={TextVariant.headingMd} align={TEXT_ALIGN.CENTER}>
+          {t('desktopPageTitle')}
+        </Text>
+        <Text marginTop={2} align={TEXT_ALIGN.CENTER}>
+          {t('desktopPageSubTitle')}
+        </Text>
         <Box
-          display={DISPLAY.FLEX}
-          alignItems={AlignItems.center}
-          textAlign={TEXT_ALIGN.CENTER}
-          flexDirection={FLEX_DIRECTION.COLUMN}
-          marginLeft={6}
-          marginRight={6}
+          marginBottom={6}
+          marginTop={6}
+          className="desktop-pairing__clickable"
+          onClick={() => {
+            handleCopy(otp);
+          }}
+          data-testid="desktop-pairing-otp-content"
         >
           <Tooltip
             wrapperClassName="desktop-pairing__tooltip-wrapper"
             position="top"
             title={copied ? t('copiedExclamation') : t('copyToClipboard')}
           >
-            <Typography
+            <Text
               align={TEXT_ALIGN.CENTER}
+              variant={TextVariant.displayMd}
               className="desktop-pairing__otp"
             >
               {otp}
-            </Typography>
+            </Text>
           </Tooltip>
+          <Box
+            display={DISPLAY.FLEX}
+            alignItems={AlignItems.center}
+            justifyContent={JustifyContent.center}
+            marginTop={4}
+            marginBottom={6}
+          >
+            <Text
+              className="desktop-pairing__countdown-timer"
+              variant={TextVariant.paragraph}
+              align={TEXT_ALIGN.CENTER}
+              backgroundColor={BackgroundColor.backgroundDefault}
+              borderRadius={BorderRadius.XL}
+              padding={2}
+            >
+              {t('desktopPairingExpireMessage', [
+                <span
+                  className="desktop-pairing__countdown-timer-seconds"
+                  key={1}
+                >
+                  {getExpireDuration()}
+                </span>,
+              ])}
+            </Text>
+          </Box>
+          <Text align={TEXT_ALIGN.CENTER} variant={TextVariant.bodySm}>
+            {t('desktopPageDescription')}
+          </Text>
         </Box>
-
-        <Typography
-          variant={TypographyVariant.paragraph}
-          align={TEXT_ALIGN.CENTER}
-          className="desktop-pairing__countdown-timer"
-        >
-          {t('desktopPairingExpireMessage', [
-            <span className="desktop-pairing__countdown-timer-seconds" key={1}>
-              {getExpireDuration()}
-            </span>,
-          ])}
-        </Typography>
-        <div className="desktop-pairing__description">
-          {t('desktopPageDescription')}
-        </div>
-      </div>
+      </>
     );
   };
 
   const renderFooter = () => {
     return (
-      <div className="desktop-pairing__buttons">
+      <Box>
         <Button
           type="primary"
           rounded
```

Comment:
Can remove these props as they are for the old component. 
```suggestion
```

Reply:
Fixed